internal/putcbblockstore.go

```diff
@@ -0,0 +1,108 @@
+package internal
```

Comment:
I don't think these should be at the top-level, they're really `internal` to the `cmd` library and should live there I think

Reply:
agree these feel internal to cmd (all the files here) so agree with @rvagg about the move